app/services/operations/list_applications.rb

```diff
@@ -23,13 +28,14 @@ def call
     private
 
     attr_reader :status, :office_code,
-                :page, :per_page, :sort_direction
+                :page, :per_page, :sort_direction, :order
 
     def query
       scope = @scope
 
       scope = scope.by_status(status) if status.present?
       scope = scope.by_office(office_code) if office_code.present?
+      scope = scope.by_applicant_name(sort_direction) if order.present? && order == 'applicant_name'
```

Comment:
Maybe is best to separate the filtering (scoping) operations and the sorting (order and direction) operations?
Here it seems we are starting to mix both.